tests/robot-tests/tests/admin/bau/invite_new_users.robot

```diff
@@ -0,0 +1,118 @@
+*** Settings ***
```

Comment:
Would suggest adding `%{WAIT_SMALL}` to things such as `user waits until h1 is visible` in order to prevent transient issues with tests running in CI

Reply:
If we should be applying `${WAIT_SMALL}` to all instances of `user waits until h1 is visible, I've changed the default wait time of that keyword to `${WAIT_SMALL}`, and then removed `${WAIT_SMALL}` when it's provided as an argument against `user waits until h1 is visible`.

Currently, it defaults to a wait of length `${timeout}` - not sure how long that is in comparison to `${WAIT_SMALL}`